joern-cli/frontends/php2cpg/src/test/scala/io/joern/php2cpg/querying/CallTests.scala

```diff
@@ -72,7 +72,7 @@ class CallTests extends PhpCode2CpgFixture {
     "have the correct method node defined" in {
       inside(cpg.call.l) { case List(fooCall) =>
         fooCall.name shouldBe "foo"
-        fooCall.methodFullName shouldBe s"Foo::foo"
+        fooCall.methodFullName shouldBe s"Foo.<class>::foo"
```

Comment:
Please remove the `.` as separator from the naming scheme.

Reply:
I find the `<class>` suffix to be unclear because i always wonder: "Is this the type of the Foo class object or the type of the Foo class objects. If there is no php specific name for this or if `<class>` is the php specific name, please name this `<metaclass>` at least that is unabiguous.